card_house/db/cards_db_columns.py

```diff
@@ -0,0 +1,6 @@
+class CardsDBColumns:
```

Comment:
Is there auto-fill for values of enums in python (C# has one ;) )? 
I think it would be better if there is, so if we'll add a new enum in the middle we won't have to shift everything...